sources/packages/web/src/composables/useOffering.ts

```diff
@@ -16,5 +16,48 @@ export function useOffering() {
     }
   };
 
-  return { mapOfferingChipStatus };
+  /**
+   * Maps the offering intensity to an user friendly string representation.
+   * @param offeringIntensity intensity of the offering.
+   * @returns user friendly string representation of the offering intensity.
+   */
+  const mapOfferingIntensity = (
+    offeringIntensity: OfferingIntensity,
+  ): string => {
+    switch (offeringIntensity) {
+      case OfferingIntensity.partTime:
+        return "Part-time";
+      case OfferingIntensity.fullTime:
+        return "Full-time";
+      default:
+        return "Unknown";
+    }
+  };
+
+  /**
+   * Maps the offering intensities to an user friendly string representation.
+   * @param allowFullTime flag to allow full time offering intensity.
+   * @returns user friendly string representations of the offering intensities.
+   */
+  const mapOfferingIntensities = (
+    allowFullTime = true,
+  ): Partial<Record<OfferingIntensity, string>> => {
+    if (allowFullTime) {
+      return {
+        [OfferingIntensity.fullTime]: mapOfferingIntensity(
+          OfferingIntensity.fullTime,
+        ),
+        [OfferingIntensity.partTime]: mapOfferingIntensity(
+          OfferingIntensity.partTime,
+        ),
+      };
+    }
+    return {
+      [OfferingIntensity.partTime]: mapOfferingIntensity(
+        OfferingIntensity.partTime,
+      ),
+    };
+  };
+
+  return { mapOfferingChipStatus, mapOfferingIntensities };
```

Comment:
Minor, `mapOfferingIntensity` can be exposed to be used. It will likely be used in the upcoming PR when we will need to render it as a read-only label inside the form.io. Does it make sense?